dts/bindings/sensor/rohm,bh1749.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
```

Comment:
This is redundant, as well as `...` at the end. See https://github.com/zephyrproject-rtos/zephyr/commit/eba81c6e54f642918b6fd8a64d50f7d444f464fa.